torchrecipes/audio/source_separation/conf/datamodule/librimix.yaml

```diff
@@ -0,0 +1 @@
+root_dir: /LibriSpeech/
```

Comment:
should this be checked-in?
Also is this supposed to be absolute path? How flexible is it?
I am not entirely sure how `torchrecipes` works, but in general, libraries cannot expect users to edit a file in it.

Reply:
Since hydra is the config system, we can't use external arguments from shell script. We can leave ``root_dir`` with blank value, but we need to put the yaml file here to notify users that they need to put the dataset path here.

Reply:
@nateanl I think you can change it to `root_dir: ???` if you want users to specify a value here.

Reply:
I think we will finally need to check in the code from the launcher side (which defines a `torchx_app` for every training recipe) and have a well-organized document showing how these apps can be launched in oss environment and giving some benchmarking numbers over some heavily used models/datasets.